app/build.gradle

```diff
@@ -9,7 +9,7 @@ apply plugin: 'kotlin-kapt'
 apply plugin: 'checkstyle'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 30
```

Comment:
Did you open the branch based on #6654 by mistake or is this intentional?

Reply:
This is intentional. I'm planning to rebase once #6654 is merged.